src/config/dynamic-theme-fixes.config

```diff
@@ -17645,6 +17645,34 @@ INVERT
 
 ================================
 
+mubi.com
+
+INVERT
+.css-jd0r3a.e1kjxayp2
+.css-k7pew6.ef3qs2t4 img
+.css-ywuexf
+.css-19064tz
+.css-1w8phor
+.css-n1xrf1
+.css-1dev2nn.e18wowia1 .star
+
+CSS
+.css-ztue73 {
+    filter: brightness(1) !important;
+}
+.css-1fdxz4i.e1f1mjpr0 svg *,
+.css-1gez225.e1t0dj1j2 svg *,
+.css-1gez225.e13p4uxl2 svg * {
+    fill: #FFFFFF !important;
+}
+.css-ker29f.ed6irnx8 {
+    filter: invert(1) hue-rotate(180deg) !important;
```

Comment:
why forcing invert this way and not include .css-ker29f.ed6irnx8 in INVERT section?